src/mqt/bench/evaluation/evaluation.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import annotations
+
+import pickle
+import sys
+from typing import TYPE_CHECKING, Any
+
+if TYPE_CHECKING or sys.version_info >= (3, 10, 0):  # pragma: no cover
+    pass
+else:
+    pass
+
+from pathlib import Path
+
+import numpy as np
+from joblib import Parallel, delayed
+from mqt.bench import utils
+from qiskit import QuantumCircuit, QuantumRegister
+from qiskit.transpiler.passes import RemoveBarriers
+
+if TYPE_CHECKING or sys.version_info >= (3, 10, 0):  # pragma: no cover
+    from importlib import resources
+else:
+    import importlib_resources as resources
+
+
+def create_statistics() -> None:
+    source_circuits_list = [file for file in Path(utils.get_qasm_output_path()).iterdir() if file.suffix == ".qasm"]
+
+    res = Parallel(n_jobs=-1, verbose=100)(delayed(evaluate_qasm_file)(filename) for filename in source_circuits_list)
+    target_dir = Path(resources.files("mqt.bench") / "evaluation/")
+    with Path(target_dir / "evaluation_data.pkl").open("wb") as f:
+        pickle.dump(res, f)
```

Comment:
I am not a big fan of dumping these results as part of the package since it needlessly increases the package size by 6.5MB. 
How long does the evaluation of these files take in general?
Is it feasible to just return `res` in this function?

Reply:
Unfortunately, the generation takes around 11h. Therefore returning `res` might not be a viable approach.

Reply:
Ok. Alternative suggestion: Can we dynamically download the `evaluation_data.pkl` from release assets just like we do with the `.zip`?

Reply:
I removed the file from the repo as discussed.